### PR TITLE
Corrected classifier code for Colab

### PR DESCRIPTION
**Pull request by take2rohit**
I have done various updates
1. Updated a README with colab notebook
2. Removed Redundant installation while using requirements.txt
3. Change the notebook on github which is same as github

### CODE REVIEW

**Comment by abhi-kumar** [comment]
1) Changes in requirements_cu9.txt would strictly install tensorflow 2.0, Monk doesn't not support tf2 as of today.

2) Could you please elaborate on the changes made for this PR : https://github.com/Tessellate-Imaging/monk_v1/pull/42/commits/56fb3f9ea51e4529b5d8dee0694e29183e55a077

3) And the colab links have been updated, right?

File: installation/Linux/requirements_cu9.txt [hunk 1]
```diff
@@ -8,10 +8,10 @@ gpustat
...
 pandas 
 GPUtil 
-keras
-tensorflow-gpu
-mxnet-cu90
-gluoncv
+keras==2.2.5 
+tensorflow-gpu==1.12.0 
+mxnet-cu90==1.5.0 
+gluoncv==0.5
```
Comment (take2rohit) [inline installation/Linux/requirements_cu9.txt]: I have restored the versions of tf

**Comment by take2rohit** [comment]
> 1. Changes in requirements_cu9.txt would strictly install tensorflow 2.0, Monk doesn't not support tf2 as of today.
> 2. Could you please elaborate on the changes made for this PR : [56fb3f9](https://github.com/Tessellate-Imaging/monk_v1/commit/56fb3f9ea51e4529b5d8dee0694e29183e55a077)
> 3. And the colab links have been updated, right?

1. In latest commit versions of tf, mxnet, etc have been restored
2. I have also added link to direct colab file in README
3. Colab links are correctly updated

**Comment by take2rohit** [comment]
As of now 56fb3f9 has code for Colab support only. This is because if we install README on our workstation it installs everything which makes the environment size large. For eg. If someone is using Pytorch then why will he install tensorflow. What I propose is a new README file for each of the frameworks.

**Comment by abhi-kumar** [comment]
@take2rohit , the changed installation instructions are very specific to colab... 

What I recommend is keeping the old instructions as it is and adding comments for what to do in case of running on colab. Person running the same on colab can comment the general instructions and uncomment the necessary ones for colab. 

Along with the same please create a requirements.txt for colab.

**Comment by take2rohit** [comment]
@abhi-kumar The colab link I have added doesn't have any need of requirements.txt. I am installing two library using inline commands. so i guess it will do the work.
I will restore the previous requirements.txt in the latest commit.

**Comment by take2rohit** [comment]
Requirements have been restored in 0e4582c

**Comment by abhi-kumar** [comment]
Exactly... the installation instructions on the notebooks have to be general... not specific to colab alone
Like this on below
Screenshot(65)

The installation will not be sufficient if someone is not running the notebook on colab.

**Pull request closed without merging**